packages/snap-preact-components/src/components/Organisms/Sidebar/Sidebar.tsx

```diff
@@ -101,7 +101,7 @@ export const Sidebar = observer((properties: SidebarProps): JSX.Element => {
 	return (
 		<CacheProvider>
 			<div {...styling} className={classnames('ss__sidebar')}>
-				{!hideTitle && <String className="ss--sidebar-title" content={subProps.String.content as string} {...subProps.String} />}
+				{!hideTitle && <Element className="ss--sidebar-title" type={'span'} {...subProps.Element} />}
```

Comment:
Any reason we are using Element here? Could we just use an H tag instead?

Reply:
Yea, I think the usage here should probably be removed, since it is unnecessary.